codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectAggregator.java

```diff
@@ -113,6 +121,15 @@ public final void setMaxCumulationBufferComponents(int maxCumulationBufferCompon
         }
     }
 
+    /**
+     * Called when a 100-continue response is requested, but the content length is too large.
+     * If this method return {@code true} then the connection will be closed. otherwise the connection
+     * will remain open and data will be consumed and discarded until the next request is received.
+     */
+    protected boolean closeOnExpectationFailed() {
```

Comment:
@normanmaurer - I thought about that, but decided not to have an extra variable stick hanging around for a condition that is a "boundary" condition for most users (just my guess).  I'm don't have a strong preference either way...WDYT?